meshdb/meshdb.go

```diff
@@ -67,6 +78,12 @@ type OrdersCollection struct {
 	IsRemovedIndex                       *db.Index
 }
 
+// MetadataCollection represents a DB collection used to store instance metadata
+type MetadataCollection struct {
+	*db.Collection
+	EthereumNetworkIDIndex *db.Index
```

Comment:
As mentioned before, we don't need to use an Index. We can simply remove this part of the code.

Reply:
Great -- seemed like overkill as I was doing it, but was closely modeling the new collection after existing ones.